aspnet/fundamentals/static-files.rst

```diff
@@ -6,7 +6,7 @@ By `Rick Anderson`_
 
 Static files, such as HTML, CSS, image, and JavaScript, are assets that an ASP.NET Core app can serve directly to clients.
 
-`View or download sample code <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/static-files/sample>`__
+`View or download sample code <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/static-files/sample>`_s_
```

Comment:
Oops, I think I was typing too fast when doing a search and accidentally added that in! I ended up just deleting this pull request and making a new one: #1589